opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -83,3 +313,75 @@ impl PushMetricsExporter for MetricsExporter {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use opentelemetry::{metrics::MeterProvider as _, KeyValue};
+    use opentelemetry_sdk::{
+        metrics::{PeriodicReader, SdkMeterProvider},
+        runtime, Resource,
+    };
+
+    use crate::exporter::MAX_EVENT_SIZE;
+    use crate::MetricsExporter; // Ensure this references the correct exporter module
+
+    #[tokio::test(flavor = "multi_thread")]
+    async fn emit_metrics_that_combined_exceed_max_event_size() {
```

Comment:
since we can't run this on CI, lets ignore them here, and use a machine with user-events available to manually run these tests?

Reply:
yes, removed the tests for now. I was planning to do some mock (so kept it in draft)., but this doesn't seem easy.